src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -325,5 +335,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
         name: 'We’re sorry. We’ve run into a problem',
       }),
     ).to.be.ok;
+
+    expect(document.activeElement).to.have.tagName('h1');
```

Comment:
I think it would be safer to wrap these active element checks in a `waitFor`, like you've done for the document title in places, unless there's already a waitFor above it for the document title. Sometimes we get flaky tests when we don't wait for these active element and document title checks.